content/articles/email-forwarding.md

```diff
@@ -16,35 +16,34 @@ categories:
 
 Email forwarding is enabled and billed monthly on a per-domain basis.
 
-The number of forwarded messages and email forwards allowed per month is based on your plan. [See the differences between plans](/articles/dnsimple-plans/).
+The number of forwarded messages and email forwards allowed per month is based on your plan. [See the differences between plans](https://dnsimple.com/pricing).
 
 ## Enabling email forwarding
 
-To enable email forwarding, visit a domain's manage page and click on the **Email forwarding** menu on the sidebar.
+1. Choose the relevant domain from your Domain List.
+2. Click the **Email Forwarding** tab on the left side. 
 
-![Email forwarding](/files/email-forwarding.png)
+<!--- needs screenshot -->
 
-Create an email forward to enable the service.
-
-When you enable email forwarding, we will automatically add the various DNS records needed to support forwarding.
+3. Create an email forward to enable the service.
 
+When you enable email forwarding, DNSimple will automatically add the various DNS records needed to support forwarding.
 
 ## Creating an email forward
 
 <div class="section-steps" markdown="1">
 ##### Adding an email forward from the dashboard
 
-1.  Log in to DNSimple with your user credentials.
 1.  If you have more than one account, select the relevant one.
 1.  Click <label>Add</label> from the Dashboard, and choose <label>Email forward</label> from the provided options.
   ![Adding an email forward](/files/add-button-email-forward.png)
 </div>
 
-To create an email forward you need to specify the **From** and **To** fields.
+To create an email forward, specify the **From** and **To** fields.
 
-![Email forwarding creation](/files/email-forwarding-setup.png)
+<!--- needs screenshot -->
```

Comment:
https://drive.google.com/file/d/1KS-SG2RwgjFr-vTYWFCoRjgM5s8C31tY/view?usp=sharing

Reply:
@dperrymorrow this one wasn't added